services-js/registry-certs/client/birth/QuestionsPage.tsx

```diff
@@ -147,15 +161,60 @@ export default class QuestionsPage extends React.Component<Props, State> {
 
     // Ensure we cannot go back any further than the first question.
     const newIndex = currentIndex > 0 ? currentIndex - 1 : 0;
+
     Router.push(`/birth?step=${steps[newIndex]}`);
   };
 
+  private gaEventOptionsLabel = (): string => {
+    const { currentStep } = this.props;
+    const { requestInformation } = this.props.birthCertificateRequest;
+
+    if (currentStep === 'forWhom') {
+      return `ordering for ${orderingFor(requestInformation)}`;
+    } else if (currentStep === 'bornInBoston') {
+      return inBoston(requestInformation);
+    } else if (currentStep === 'parentalInformation') {
+      return `parents married: ${requestInformation.parentsMarried}`;
+    } else {
+      return '';
+    }
+
+    function orderingFor(requestInformation): string {
+      const { forSelf, howRelated } = requestInformation;
+
+      if (forSelf) {
+        return 'self';
+      } else {
+        return howRelated as string;
+      }
+    }
+
+    function inBoston(requestInformation): string {
+      const { bornInBoston, parentsLivedInBoston } = requestInformation;
+
+      let result = `born in Boston: ${bornInBoston}`;
+
+      if (bornInBoston !== 'yes') {
+        result += `; parents lived in Boston: ${parentsLivedInBoston}`;
+      }
+
+      return result;
+    }
+  };
+
+  gaSendEvent = (label: string): void => {
+    this.props.siteAnalytics.sendEvent('click', {
```

Comment:
I'd recommend changing this so that you have separate actions for each kind of question, and then use the label as the answer to that question.

That way you can filter down per question.

Reply:
Awesome, this syncs with where my head is at.